diff -aurN old/Makefile new/Makefile
--- old/Makefile	2014-12-01 18:39:02.000000000 +0800
+++ new/Makefile	2019-06-03 10:48:01.390267265 +0800
@@ -5,8 +5,8 @@
 INSTALL_DATA ?= install
 
 CC?=gcc
-CFLAGS?=-O2 -Wall -g -D_FORTIFY_SOURCE=2 -fstack-protector -fPIC
-LDFLAGS?=-Wl,-z,now -Wl,-z,relro -Wl,-soname,libscrypt.so.0 -Wl,--version-script=libscrypt.version
+CFLAGS?=-O2 -Wall -g -D_FORTIFY_SOURCE=2 -fPIC
+LDFLAGS?=-Wl,-soname,libscrypt.so.0 -Wl,--version-script=libscrypt.version
 CFLAGS_EXTRA?=-Wl,-rpath=.
 
 all: reference
@@ -14,7 +14,7 @@
 OBJS= crypto_scrypt-nosse.o sha256.o crypto-mcf.o b64.o crypto-scrypt-saltgen.o crypto_scrypt-check.o crypto_scrypt-hash.o slowequals.o
 
 libscrypt.so.0: $(OBJS) 
-	$(CC)  $(LDFLAGS) -shared -o libscrypt.so.0  $(OBJS) -lm -lc
+	$(CC)  $(LDFLAGS) -shared -o libscrypt.so.0  $(OBJS) -lm 
 	ar rcs libscrypt.a  $(OBJS)
 
 reference: libscrypt.so.0 main.o b64.o crypto_scrypt-hexconvert.o
